schemas/package.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: package
+  title: Package
+  group: 2
+  short: These fields contain information about an installed software package.
+  description: >
+    These fields contain information about an installed software package. It contains general information about a
+    package, such as name, version or size. It also contains installation details, such as time, URL or
+    user who installed the package.
+  type: group
+  fields:
+
+    - name: name
+      level: core
```

Comment:
General note: I would put all these fields into extended for now.

Reply:
Agreed